profile/index.html

```diff
@@ -0,0 +1,109 @@
+</head>
+<body>
+    <main class="">
+        <nav class="navbar navbar-expand-lg navbar-dark bg-body-tertiary">
+            <div class="container-fluid">
+              <ul class="navbar-nav mt-4">
+                <li class="nav-item dropdown">
+                    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                        <img src="/img/perfil.png" class="rounded-circle" height="46" alt="Portrait" loading="lazy"/>
+                        <div class="navbar-nav">
+                          <a class="nav-link active" aria-current="page" href="#">Profile</a>
+                          <a class="nav-link" href="/feed/index.html">Feed</a>
+                        </div>
+                    </div>
+                </li>
+              </ul>
+            </div>
+        </nav>
+
+        <div class="m-auto col-md-8">
+            <div class="card">
+                <div class="card-body little-profile text-center">
+                    <div class="pro-img"><img src="/img/perfil.png" alt="user" height="136"></div>
+                    <h3 class="m-b-0">Ricardo Nunes</h3>
+                    <p>Web Designer &amp; Developer</p> <a href="/feed.html" class="m-t-10 waves-effect waves-dark btn btn-primary btn-md btn-rounded" data-abc="true">Follow</a>
+                    <div class="row text-center m-t-20">
+                        <div class="col-lg-4 col-md-4 m-t-20">
+                            <h3 class="m-b-0 font-light">7</h3><small>Photos</small>
+                        </div>
+                        <div class="col-lg-4 col-md-4 m-t-20">
+                            <h3 class="m-b-0 font-light">34K</h3><small>Followers</small>
+                        </div>
+                        <div class="col-lg-4 col-md-4 m-t-20">
+                            <h3 class="m-b-0 font-light">1454</h3><small>Following</small>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+       
+        <hr size="3" width="100%" color="#ffffff" style="margin: 0%;">
+        <ul class="nav nav-tabs nav-feed">
+            <li class="nav-item">
+                <a class="nav-link active" id="home-tab" data-bs-toggle="tab" data-bs-target="#home" type="button" role="tab" aria-controls="home" aria-selected="true">Photos</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" id="profile-tab" data-bs-toggle="tab" data-bs-target="#profile" type="button" role="tab" aria-controls="profile" aria-selected="false">Tagget</a>
+            </li>
+        </ul>
+
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-md-8 mx-auto">
+                    <div class="tab-pane fade show active" id="home" role="tabpanel" aria-labelledby="home-tab">
+
+                        <div class="row" style="padding-bottom: 25px;">
```

Comment:
I think you should use pb instead of inline styling and rather change the values or add new ones in the variables.scss file. I may not see the big picture here but that may be easier to maintain in the long run.